main.js

```diff
@@ -89,36 +87,71 @@ class Timer {
 
 class DOM_Manager {
     constructor() {
-        this.cardsCountVar = document.getElementsByClassName("card").length;
         this.clickListenerBinded = this.clickListener.bind(this);
-        this.imgs = [];
-        for(let i=0;i<this.cardsCountVar;i++){
-            this.imgs.push(document.getElementById("img"+i));
-        }
+        this.submitOnClickBinded = this.submitOnClick.bind(this);
+        this.game=new Game();
+        document.getElementById("Easy").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Medium").addEventListener("change", this.difficultOnChange);
+        document.getElementById("Hard").addEventListener("change", this.difficultOnChange);
+        document.getElementById("submit").addEventListener("click", this.submitOnClickBinded);
+    }
+    difficultOnChange() {
+        document.getElementById("submit").disabled = false;
     }
-    setListeners(fn) {
-        this.callBackFn = fn
-        for(let img of this.imgs) {
-            img.addEventListener('click', this.clickListenerBinded);
+    submitOnClick() {
+        const selectedDiff = document.querySelectorAll('input[name="difficulty"]:checked');
+        this.cardsCount = parseInt(selectedDiff[0].value);
+        document.getElementById("modalContainer").classList.add('hidden');
+        const cards = this.createCards();
+        this.game.start(cards);
+    }
+    createCards() {
+        const fragment=document.createDocumentFragment();
+        const cards = [];
+        for(let i = 0;i<this.cardsCount;i++) {

```

Comment:
I suppose you really like `for` loop :)
Consider this:
```js
const cards = Array(this.cardsCount).fill(0)
  .map((_, index) => new Card(index, this.cardsCount, this.clickListenerBinded));

cards.forEach(card => fragment.appendChild(card.img));
```
You don't have to change your code. Just wanted to show you how awesome arrays can be